### DISCUSSION TITLE
Task List issue creation from a Discussion

### DISCUSSION THREAD

**Discussion by hanselabreu** [community/community#4274] [Projects and Issues]
Great job on the new feature! Definitely comes in handy when managing a project. 

I would love to have the same ability to create issues from a task list on a discussion. This would especially be useful using a discussion to track a roadmap.

**Comment by labrose** [comment]
Glad you find it useful!

When it comes to creating issues from your discussion, are you looking mostly for the same quick issue creation or would you also expect the created issues to be related in some way (ie the discussion is "tracking" the roadmap items or itself might be converted to an issue)?

**Reply by hanselabreu** [reply]
For me personally the quick issue creation would suffice. However, the second option you described would be amazing 🤯
